avogadro/quantumio/orca.cpp

```diff
@@ -332,13 +336,23 @@ void ORCAOutput::processLine(std::istream& in, GaussianSet* basis)
           break;
         list = Core::split(key, ' ');
         vector<int> modeIndex;
+        bool invalid_index = false;
         while (!key.empty()) {
           // first we get a set of column numbers
           // e.g. 1  2  3  4  5  6  7  8  9 10
           modeIndex.clear();
           for (unsigned int i = 0; i < list.size(); i++) {
-            modeIndex.push_back(Core::lexicalCast<int>(list[i]));
+            int index = Core::lexicalCast<int>(list[i]);
```

Comment:
Shouldn't we just declare (index) as `std::size_t` here?

Reply:
Yes, I just kept it as it was before.